src/patterns/language-selection/index.njk

```diff
@@ -0,0 +1,42 @@
+---
+title: Change Language
+group: Help users to...
+---
+{% from "views/partials/example/_macro.njk" import patternlibExample %}
+
+## When to use this pattern
+
+This pattern should be if you service is available in multiple languages.
+
+## How to use this pattern
+
+Language selection is achieved by rendering a language selector component in the header element, and a list of available lanauges in the footer. The header lanauge selector is for viewports larger than mobile, and the footer language selector is for viewports smaller than tablet.
+
+If there are 3 or less available languages, the header language picker will render two links, otherwise a dropdown menu will be renedered.
```

Comment:
```suggestion
If there are 3 or less available languages, the header language picker will render one or two links, otherwise a select component will be rendered.
```